src/main/java/com/algorand/algosdk/templates/ContractTemplate.java

```diff
@@ -14,44 +14,66 @@
 
 public class ContractTemplate {
 
-	public enum DataType {
-		BASE64, ADDRESS, INT;
+	/**
+	 * Values are appended to a program at specific offsets. Depending on the type the source template has placeholders
```

Comment:
Do you want to mention here something that these are not cross-SDK compatible? That each binary works only within the SDK it was designed for, and that the placeholder sizes are also SDK specific?

Reply:
Good point, clarification added.